src/Samples/TestEmbedded/build.cmd

```diff
@@ -22,7 +22,7 @@ if exist %nuget_dir%\nuget.exe (
   )
 )
 if not exist %nuget_dir% md %nuget_dir%
-if not exist %nuget_dir%\nuget.exe powershell -Command "Invoke-WebRequest https://dist.nuget.org/win-x86-commandline/v5.8.0-preview.2/nuget.exe -OutFile %nuget_dir%\nuget.exe"
+if not exist %nuget_dir%\nuget.exe powershell -Command "Invoke-WebRequest https://dist.nuget.org/win-x86-commandline/v6.3.0/nuget.exe -OutFile %nuget_dir%\nuget.exe"
```

Comment:
Thanks for getting this done.  I assume you want 6.3.1 here to reflect the version in the readme.